change/@uifabric-merge-styles-7568f8f0-b308-4ee8-922e-a0afc4c3f931.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add some logic to stylesheet that will do a simple append instead of using insert before if the reference node isn't actually a child of the document head.",
```

Comment:
```suggestion
  "comment": "fix: Add some logic to stylesheet that will do a simple append instead of using insert before if the reference node isn't actually a child of the document head.",
```